bin/all_sky_search/pycbc_sngls_statmap

```diff
@@ -49,6 +49,12 @@ parser.add_argument('--veto-window', type=float, default=.1,
                     help='Time around each zerolag trigger to window out '
                          '[default=.1s]')
 significance.insert_significance_option_group(parser)
+parser.add_argument('--hierarchical-removal-ifar-threshold',
+                    type=float, default=0.5,
+                    help="Threshold to remove foreground triggers with "
+                         "IFAR (years) above this value from the exclusive "
+                         "background, and for hierarchical removal "
+                         "[default=0.5yr]")
```

Comment:
```suggestion
                    help="Threshold to remove foreground triggers with "
                         "IFAR (years) above this value during "
                         "hierarchical removal [default=0.5yr]")
```

Reply:
-> 'Threshold to hierarchically remove foreground triggers with IFAR (years) above this value'